forage-android/src/main/java/com/joinforage/forage/android/Utils.kt

```diff
@@ -24,28 +23,3 @@ internal fun TypedArray.getBoxCornerRadius(styleIndex: Int, defaultBoxCornerRadi
     val styledBoxCornerRadius = getDimension(styleIndex, 0f)
     return if (styledBoxCornerRadius == 0f) defaultBoxCornerRadius else styledBoxCornerRadius
 }
-
-// This extension splits the path by "/" and adds each segment individually to the path.
-// This is to prevent the URL from getting corrupted through internal OKHttp URL encoding.
-internal fun HttpUrl.Builder.addPathSegmentsSafe(path: String): HttpUrl.Builder {
-    path.split("/").forEach { segment ->
-        if (segment.isNotEmpty()) {
-            this.addPathSegment(segment)
-        }
-    }
-    return this
-}
-
-/**
- * [JSONObject.optString] has trouble falling back to `null` and seems to fallback to `"null"` (string) instead
- */
-internal fun JSONObject.getStringOrNull(fieldName: String): String? {
-    if (!has(fieldName) || isNull(fieldName)) {
-        return null
-    }
-    return optString(fieldName)
-}
-
-internal fun JSONObject.hasNonNull(fieldName: String): Boolean {
-    return has(fieldName) && !isNull(fieldName)
-}
```

Comment:
This is necessary for ecommerce

Reply:
Agreed! Ended up moving these facilities to the `core` module in the [next PR in the stack.](https://app.graphite.dev/github/pr/teamforage/forage-android-sdk/247/Organize-all-files-into-core-and-ecom-packages?panel=files#comment-PRRC_kwDOIQqcBs5fMpUJ). Lmk whether that link addresses this